internal/controllers/machine/machine_controller.go

```diff
@@ -460,6 +503,8 @@ func (r *Reconciler) reconcileDelete(ctx context.Context, s *scope) (ctrl.Result
 			if conditions.Get(m, clusterv1.VolumeDetachSucceededCondition) == nil {
 				conditions.MarkFalse(m, clusterv1.VolumeDetachSucceededCondition, clusterv1.WaitingForVolumeDetachReason, clusterv1.ConditionSeverityInfo, "Waiting for node volumes to be detached")
 			}
+			s.deletingReason = clusterv1.MachineDeletingWaitingForVolumeDetachV1Beta2Reason
+			s.deletingMessage = ""
```

Comment:
Am I wrong in assuming we will get more info in this message when @chrischdi PR merges?

Reply:
Also, let's add the start time here, in case there are code branches not setting the condition below

Reply:
> Am I wrong in assuming we will get more info in this message when @chrischdi PR merges?

Not this one here, but I think the one in shouldWaitForNodeVolumes

> Also, let's add the start time here, in case there are code branches not setting the condition below

done